hypergpufunctions.h

```diff
@@ -34,18 +34,20 @@ class HyperFunctionsGPU : public HyperFunctions {
     int *d_img_array,  *d_out, *d_ref_spectrum; 
     int block_size = 512;
     int grid_size ;
+    int *img_array_base;
     
     
     void spec_sim_GPU();
     void allocate_memory(int* img_array);
     void deallocate_memory( );
     int* mat_to_oneD_array_parallel_parent();
-    void oneD_array_to_mat(int* img_array);
+    void oneD_array_to_mat(int* img_array, int rows, int cols);
```

Comment:
is it necessary to pass rows, cols? Can it be figured out from inside function?

Reply:
fixed